toptal/picasso
# fix: prompt modal submit button when loading is not disabled

### Description

While the PromptModal submit button is loading, it is no longer disabled. See more details on https://github.com/toptal/picasso/issues/1425.

### How to test

- To go [Temploy](https://picasso.toptal.net/fix-prompt-modal-submit-button-loading-state/?path=/story/components-folder--promptmodal#default)
- Click on "Open prompt" button
- Click on "Ok" button, and note that while the "Ok" button is loading, it's still enabled.  

### Review
